src/components/KeyboardSpacer/BaseKeyboardSpacer.js

```diff
@@ -0,0 +1,107 @@
+import React, {PureComponent} from 'react';
+import {
+    Dimensions, Keyboard, LayoutAnimation, View,
+} from 'react-native';
+import * as StyleUtils from '../../styles/StyleUtils';
+import {propTypes, defaultProps} from './BaseKeyboardSpacerPropTypes';
+
+const defaultAnimation = {
+    duration: 500,
+    create: {
+        duration: 300,
+        type: LayoutAnimation.Types.easeInEaseOut,
+        property: LayoutAnimation.Properties.opacity,
+    },
+    update: {
+        type: LayoutAnimation.Types.spring,
+        springDamping: 200,
+    },
+};
+
+class BaseKeyboardSpacer extends PureComponent {
+    constructor(props) {
+        super(props);
+        this.state = {
+            keyboardSpace: 0,
+        };
+        this.keyboardListeners = null;
+        this.updateKeyboardSpace = this.updateKeyboardSpace.bind(this);
+        this.resetKeyboardSpace = this.resetKeyboardSpace.bind(this);
+    }
+
+    componentDidMount() {
+        const updateListener = this.props.keyboardShowMethod;
+        const resetListener = this.props.keyboardHideMethod;
+        this.keyboardListeners = [
+            Keyboard.addListener(updateListener, this.updateKeyboardSpace),
+            Keyboard.addListener(resetListener, this.resetKeyboardSpace),
+        ];
+    }
+
+    componentWillUnmount() {
+        this.keyboardListeners.forEach(listener => listener.remove());
+    }
+
+    /**
+     * Update the height of Keyboard View.
+     *
+     * @param {Object} [event] - A Keyboard Event.
+     */
+    updateKeyboardSpace(event) {
+        if (!event.endCoordinates) {
+            return;
+        }
+
+        let animationConfig = defaultAnimation;
+        if (this.props.iOSAnimated) {
```

Comment:
Sure, let me check on that

Reply:
<img width="697" alt="Screenshot 2022-08-11 at 10 50 22 PM" src="https://user-images.githubusercontent.com/47522946/184194741-cf46c75a-3d30-4824-bb13-a9982bb968dd.png">

I did check for this one, we can't move animation to platform files as we are taking events data from keyboard listeners

Reply:
need to check about this one

Reply:
@rushatgabhane actually, we can remove this animation thing since I tested it and came to know it has no effect 

**Without Animation**

https://user-images.githubusercontent.com/47522946/184830288-ddba0ab1-7f5a-46b4-9cb7-069d2bb800e7.mov

**With Animation**

https://user-images.githubusercontent.com/47522946/184830324-e4b0cd13-1ddb-48f0-8f68-c95bce7ab149.mov

**Note: Debug build are slower in latest version**

Reply:
Removing the animation seems fine to me if it does not show up really

Reply:
we can keep the default animation for both platforms above video we have default animation on iOS

Reply:
but I think we need to keep defaultAnimation as it feels pretty good on both platforms

Reply:
@rushatgabhane have removed iOS specific animation